charger/easee.go

```diff
@@ -379,22 +388,108 @@ func (c *Easee) Enable(enable bool) error {
 		}
 
 		uri := fmt.Sprintf("%s/chargers/%s/settings", easee.API, c.charger)
-		resp, err := c.Post(uri, request.JSONContent, request.MarshalJSON(data))
-		if err != nil {
+		if err := c.postJSONAndWait(uri, request.MarshalJSON(data)); err != nil {
 			return err
 		}
-		resp.Body.Close()
 	}
 
 	// resume/stop charger
 	action := easee.ChargePause
 	if enable {
 		action = easee.ChargeResume
 	}
+
 	uri := fmt.Sprintf("%s/chargers/%s/commands/%s", easee.API, c.charger, action)
-	_, err := c.Post(uri, request.JSONContent, nil)
+	if err := c.postJSONAndWait(uri, nil); err != nil {
+		return err
+	}
 
-	return err
+	if enable {
+		// reset currents after enable, as easee automatically resets to maxA
+		return c.MaxCurrent(int64(c.current))
+	}
+
+	return nil
+}
+
+// posts JSON to the Easee API endpoint and waits for the async response
+func (c *Easee) postJSONAndWait(uri string, data io.ReadSeeker) error {
```

Comment:
```suggestion
func (c *Easee) postJSONAndWait(uri string, data any) error {
```

...und dann das MarshalJson- falls data != nil- hier drin machen, spart Dir das seek.